server/chat-plugins/poll.ts

```diff
@@ -543,8 +543,9 @@ export const commands: ChatCommands = {
 export const pages: PageTable = {
 	pollqueue(args, user) {
 		this.extractRoom();
-		const room = Rooms.get(args[0]) as ChatRoom | GameRoom;
-		let buf = `<div class = "pad"><strong>Queued polls:</strong>`;
+		const room = this.room;
+		if (!room) return `<div class="pad"><h2>Specify a room.</h2></div>`;
```

Comment:
We should probably change it so `extractRoom` can throw an error message for requiring a room.